lxd/networks.go

```diff
@@ -806,6 +825,21 @@ func networkShutdown(s *state.State) error {
 	return nil
 }
 
+func networkStateGet(d *Daemon, r *http.Request) Response {
+	name := mux.Vars(r)["name"]
+
+	// Get some information
+	osInfo, _ := net.InterfaceByName(name)
+	_, dbInfo, _ := d.cluster.NetworkGet(name)
```

Comment:
Ignoring errors is safe here?

Reply:
Yeah, that's normal. Networks can be defined on the system (osInfo) or in the database (dbInfo) or both.
Them being missing in one or the other is fine, we only fail if both are missing.